pyproject.toml

```diff
@@ -113,7 +115,7 @@ app_package_subdir = 'app'
 #docs_package_version = '2020.7.11'
 
 [ci.app.tutorials.video]
-fps = 16                # Frames per Second
+fps = 25                # Frames per Second
```

Comment:
this will double the size of the videos. Do we have enough room to store them all?

Reply:
The only video tutorial that currently exists is about 400 kB, so that's not a problem. We'll see later when more video tutorials are created.